src/mobu/services/business/nublado.py

```diff
@@ -188,6 +188,7 @@ async def execution_idle(self) -> bool:
             return await self.pause(self.options.execution_idle_time)
 
     async def shutdown(self) -> None:
+        await self.hub_login()
```

Comment:
What about putting a `hub_login` call in `delete_lab`?  Tradeoff is that we may call it a few extra times.

Reply:
I think I prefer it be explicit.